### ISSUE TITLE
Bump leaflet from 1.3.1 to 1.6.0

### ISSUE THREAD

**Issue by crisner** [publiclab/leaflet-environmental-layers#355]
Bringing to notice that we still seem to be using an older version of leaflet whereas plots2 uses the latest version(1.6.0).

**Comment by sagarpreet-chadha** [comment]
🎉 We need to upgrade. Thanks!
Do you think you can make a FTO for this? (Remember to add steps to run` yarn install` after changing package number). Thanks!

**Comment by emilyashley** [comment]
Ooh good call.

**Comment by crisner** [comment]
> Do you think you can make a FTO for this? (Remember to add steps to run yarn install after changing package number). 

Yes. I can make an FTO out of this. We'll need to run `npm install` here in LEL right? Is this to generate the package-lock.json and commit that file too?

**Comment by jshreyans** [comment]
Hi! Can I work on this issue? I'm new to the codebase and would love to start contributing.

**Comment by crisner** [comment]
@jshreyans, sure! Please go ahead! As mentioned above remember to run `npm install` after changing the package number. 😃 Thanks!

**Comment by neelesh17** [comment]
Hi, May I work on this Issue? I,m new to open source contribution and would love to get started.

**Comment by jshreyans** [comment]
@crisner Thanks a lot! I'll send a PR as soon as possible.

**Comment by crisner** [comment]
@neelesh17, this issue has already been claimed. Feel free to look at other first-timers-only issues in this repo or here: https://code.publiclab.org/. Thanks!

**Comment by Hina-softwareEngineer** [comment]
@crisner Is this issue resolved? Can I work on it? I am new to open source and want to contribute to javascript projects.
